brainzutils/musicbrainz_db/serialize.py

```diff
@@ -197,10 +198,17 @@ def serialize_url(url, includes=None):
     return data
 
 
+def serialize_editors(editor, includes=None):
+    data = row2dict(editor, exclude_pk=True, exclude={'password', 'ha1'})
```

Comment:
nitpick, but just returning the result of row2dict here without assigning it to data would be cleaner.

Reply:
Yeah, I intentionally left it like this because the includes will have to be added to data before it's returned. I added a comment explaining that.